packages/api-server/src/rate-limit.ts

```diff
@@ -88,6 +101,35 @@ export async function rateLimit(
   return isBan;
 }
 
+export async function _rateLimit(
```

Comment:
Rename this method without `_` prefix?

Reply:
we have a method `rateLimit` already, this two can be refactor in the future

Reply:
I means don't start with `_`, in our eslint rule, `_` prefix means ignore

Reply:
updated